bundles/specmate-integration-test/src/com/specmate/test/integration/AttributeValidationServiceTest.java

```diff
@@ -0,0 +1,188 @@
+package com.specmate.test.integration;
+
+import static org.junit.Assert.fail;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.util.tracker.ServiceTracker;
+
+import com.mifmif.common.regex.Generex;
+import com.specmate.common.SpecmateException;
+import com.specmate.common.SpecmateValidationException;
+import com.specmate.model.base.BaseFactory;
+import com.specmate.model.base.Folder;
+import com.specmate.model.support.api.IAttributeValidationService;
+
+public class AttributeValidationServiceTest {
```

Comment:
We could implement this test as a regular Junit test, which would run faster as an integration test as we do not need to start the whole framework